src/reducers/SearchSlice.tsx

```diff
@@ -64,23 +65,29 @@ const search = createSlice({
       })
       .addCase(fetchRecentRevisions.rejected, (state, action) => {
         state.inputError = true;
-        state.inputHelperText = action.payload;
+        state.inputHelperText = action.payload
+          ? action.payload
+          : 'An error has occurred';
```

Comment:
I had to check for undefined here, or else I would get a type error of `string | undefined`

Reply:
That's common when using type systems. Sometimes you _know_ something can't happen, but you still need to check it. I believe this is a small price to pay for the added type safety.